airflow/cli/commands/task_command.py

```diff
@@ -80,6 +80,7 @@ def _run_task_by_executor(args, dag, ti):
             with create_session() as session:
                 pickle = DagPickle(dag)
                 session.add(pickle)
+                session.commit()
                 pickle_id = pickle.id
                 # TODO: This should be written to a log
                 print(f'Pickled dag {dag} as pickle_id: {pickle_id}')
```

Comment:
```suggestion
            pickle_id = pickle.id
            # TODO: This should be written to a log
            print(f'Pickled dag {dag} as pickle_id: {pickle_id}')
```

Reply:
@ashb Ahh! Thanks for your inputs! I made the changes as per your suggestions.

earlier I was referring to below code
https://github.com/apache/airflow/blob/3711a29e5e253ede4ab12efe5990e83240e8a9a9/airflow/jobs/backfill_job.py#L782-L785